src/util/duration.ts

```diff
@@ -3,20 +3,20 @@ import { parse } from "tinyduration";
 export function formatDuration(isoDuration: string): string {
   const parsedTime = parse(isoDuration);
 
-  const hours = parsedTime.hours ? `${parsedTime.hours}:` : "";
-  let minutes = "0:";
-  if (parsedTime.minutes) {
-    minutes = `${parsedTime.minutes}:`;
-    if (parsedTime.hours && parsedTime.minutes < 10) {
-      minutes = `0${minutes}`;
-    }
+  const parsedHours = parsedTime.hours || 0;
+  const parsedMinutes = parsedTime.minutes || 0;
+  const parsedSeconds = parsedTime.seconds || 0;
+
+  const hours = parsedHours ? `${parsedHours}:` : "";
+
+  let minutes = `${parsedMinutes}:`;
+  if (parsedHours && parsedMinutes < 10) {
+    minutes = `0${minutes}`;
   }
-  let seconds = "00";
-  if (parsedTime.seconds) {
-    seconds = String(parsedTime.seconds);
-    if (parsedTime.minutes && parsedTime.seconds < 10) {
-      seconds = `0${seconds}`;
-    }
+
+  let seconds = `${parsedSeconds}`;
```

Comment:
use `String` instead here
```suggestion
  let seconds = String(parsedSeconds);
```